File: web/src/app/webapi/models/schedule-user-for-candidate-model.ts
/* tslint:disable */
import { UserViewModel } from './user-view-model';
import { ApprovalEventViewModel } from './approval-event-view-model';
import { CandidateViewModel } from './candidate-view-model';

/**
 */
export class ScheduleUserForCandidateModel {
    interviewScheduledDate?: string;
    scheduleUserForCandidateId?: number;
    candidateId?: string;
    userId?: string;
    scheduledOn?: string;
    approvalEventId?: number;
    interviewScheduledTime?: string;
    isFinished?: boolean;
    user?: UserViewModel;
    approvalEvent?: ApprovalEventViewModel;
    candidate?: CandidateViewModel;
}

File: api/RS.API/RS.Model/Candidate/ScheduleUserForCandidateModel.cs
using RS.ViewModel.Approval;
using RS.ViewModel.User;
using System;
using System.Collections.Generic;

namespace RS.ViewModel.Candidate
{
    public class ScheduleUserForCandidateModel
    {
        public int ScheduleUserForCandidateId { get; set; }
        public int ApprovalEventId { get; set; }
        public Guid CandidateId { get; set; }
        public Guid UserId { get; set; }
        public DateTime ScheduledOn { get; set; }
        public string InterviewScheduledDate { get; set; }
        public string InterviewScheduledTime { get; set; }
        public bool IsFinished { get; set; }
        public UserViewModel User { get; set; }
        public ApprovalEventViewModel ApprovalEvent { get; set; }
        public CandidateViewModel Candidate { get; set; }
    }
}
